src/plugins/csvstorage/README.md

```diff
@@ -0,0 +1,30 @@
+- infos = Information about the csvstorage plugin is in keys below
+- infos/author = Thomas Waser
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = storage
+- infos/placements = getstorage setstorage
+- infos/description = parses csv files
+
+## Introduction ##
+
+This plugin allowes Elektra to read and write CSV files.
+
+## Configuration ##
+
+`config/delimiter`
```

Comment:
You can leave out the `config/` because neither the user (`kdb mount`) nor the plugin need to care about the details in `system/elektra/mountpoints`